### ISSUE TITLE
[Bug] Incorrect version `4.3.3` in `openbb_platform/pyproject.toml` for release `4.3.4`

### ISSUE THREAD

**Issue by the-praxs** [OpenBB-finance/OpenBB#6885]
**Describe the bug**

@deeleeramone @IgorWounds Pretty sure this is a minor mistake for the new release (#6870) -

```toml
[tool.poetry]
name = "openbb"
version = "4.3.3"
description = "Investment research for everyone, anywhere."
```

**Comment by deeleeramone** [comment]
And this is what it is in the PR that is merging main into develop #6875:

<img width="1113" alt="Screenshot 2024-10-26 at 9 23 06 AM" src="https://github.com/user-attachments/assets/3af1e274-a758-4165-9468-b4f39ee49afe">

**Comment by deeleeramone** [comment]
and here's the diff from the PR you linked, @the-praxs:

<img width="973" alt="Screenshot 2024-10-26 at 9 26 01 AM" src="https://github.com/user-attachments/assets/0de95f28-f4ca-4756-98e1-6527c321d97e">

**Comment by the-praxs** [comment]
This is on the `develop` branch, where `main`  hasn't been merged yet (#6875). That's why it's still showing `4.3.3`.
